docs/source/development.rst

```diff
@@ -23,39 +23,6 @@ To run all tests, simply run ``pytest`` in the repository.  To run a specific te
 run ``pytest tests/test_api.py``, for example.
 
 
-CQL extension lifecycle
```

Comment:
Why are we removing this entire section? Maybe it's better for DX to remove only `Model generation` and `How to merge` and add a paragraph `How to hack the models`, what do you think?

Reply:
What do you envision as being the content of this alternative `How to hack the models` section? 

In my opinion it would likely make sense if you'd still like to recommend a workflow like the autogeneration of models using `datamodel-code-generator` and then some manual tweaking. But I don't see much value in it, as the models already exist and have modifications, so it seems more likely that further changes will not restart from scratch - in which case, it seems to me that _How to hack the models_ is not so different from _how to develop pygeoapi as a whole_.

But maybe I'm missing the point :sweat_smile: ?